mkt/extensions/tests/test_views.py

```diff
@@ -148,6 +149,11 @@ def setUp(self):
         self.list_url = reverse('api-v2:extension-list')
         self.user = UserProfile.objects.get(pk=2519)
 
+    def tearDown(self):
+        super(TestExtensionViewSetPost, self).tearDown()
+        # Explicitely delete the Extensions to clean up leftover files.
```

Comment:
With a quick skim of this commit, I don't see anywhere where you'd be creating files. Should this be there already?